test/kernel/test_autotuner.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+# mypy: ignore-errors
+# This test takes a long time to run
+import unittest
+import torch
+import pytest
+from parameterized import parameterized
+
+import logging
+import os
+
+
+class TestQuantFlow(unittest.TestCase):
+
+    def setUp(self):
+        os.environ["TORCHAO_AUTOTUNER_ENABLE"] = "1"
+
+    def tearDown(self):
+        del os.environ["TORCHAO_AUTOTUNER_ENABLE"]
+
+    @parameterized.expand(
+            [
+                ("cuda", torch.bfloat16),
+                ("cuda", torch.bfloat16),
+                # ("cpu", torch.bfloat16),
```

Comment:
nit: remove comments

Reply:
I'll turn those into TODOs. It should also work on CPU.